pkg/install/condition.go

```diff
@@ -16,13 +16,13 @@ func (i *Installer) bootstrapConfigMapReady() (bool, error) {
 }
 
 func (i *Installer) apiServersReady() (bool, error) {
-	apiserver, err := i.operatorcli.OperatorV1().KubeAPIServers().Get("cluster", metav1.GetOptions{})
```

Comment:
apiServer stopped exposing these conditions. Need to verify where it happened, but from the patterns, I see that most operators expose it via configCLI

Reply:
@sttts please can you help with this question?  In 4.3 ARO was checking the conditions on KubeAPIServer.operator.openshift.io to validate that the API servers were done rotating during install.  When did this go away and is checking the kube-apiserver ClusterOperator OK as an alternative?

Reply:
Yes, the cluster operator is the only definitive source for this. Conditions come and go. Never ever depend on certain conditions.